data/formats-data.js

```diff
@@ -115,15 +71,15 @@ let BattleFormatsData = {
 	},
 	caterpie: {
 		encounters: [
-			{"generation": 1, "level": 3, "shiny": false},
+			{"generation": 1, "level": 3},
```

Comment:
?

I forget, are events non-shiny by default, and `"shiny": false` means nothing?

Reply:
Looks like yes. I'll update the documentation.

Reply:
Sorry for late reply; yeah, it's either `true` or `1`, `false` means never